packages/zowe-explorer-api/src/profiles/ZoweExplorerZosmfApi.ts

```diff
@@ -151,6 +151,10 @@ export class ZosmfUssApi extends ZosmfApiCommon implements ZoweExplorerApi.IUss
         return zowe.Download.ussFile(this.getSession(), inputFilePath, options);
     }
 
+    public fileUtils(outputPath: string, options: object): Promise<Buffer> {
+        return zowe.Utilities.putUSSPayload(this.getSession(), outputPath, options);
+    }
```

Comment:
Although z/OSMF has a common "file utilities" endpoint for requests such as copy, chmod, move, etc, I think most other APIs like FTP or SSH would implement them as separate commands/endpoints.

Would it make sense to implement separate methods on `ZoweExplorerApi` for each action? For example:
```
public copy(outputPath: string, options: object): Promise<Buffer> {
    return zowe.Utilities.putUSSPayload(this.getSession(), outputPath, { ...options, request: "copy" });
}

public chmod(outputPath: string, options: object): Promise<Buffer> {
    return zowe.Utilities.putUSSPayload(this.getSession(), outputPath, { ...options, request: "chmod" });
}
```

Reply:
We have discussed this together outside of the PR, and agree that this convention would work better with extenders. If everyone is happy with this approach I will push up the changes that implement this.